lnst/RecipeCommon/Perf/Measurements/LinuxPerfMeasurement.py

```diff
@@ -89,7 +90,7 @@ def _get_host_perf_version(self) -> Optional[str]:
     @property
     def configurations(self) -> Iterable[tuple[Host, int, list[int]]]:
         for host in self.hosts:
-            for cpu_list_id, cpu_list in enumerate(self.linuxperf_cpus):
+            for cpu_list_id, cpu_list in enumerate(self.linuxperf_cpus[host]):
```

Comment:
I think you can merge those 2 `for`s to single one:
```python
for cpu_list_id, (host, cpu_list) in enumerate(self.linux_perf_cpus.items()):
    yield host, cpu_list_id, cpu_list
```

Reply:
Good idea. Will fix.

Reply:
Actually, the semantics of youe code is different.

Your code would generate:
```
0, host1, list1
1, host1, list2
2, host2, list1
3, host2, list2
```
My code generates:
```
0, host1, list1
1, host1, list2
0, host2, list1
1, host2, list2
```

The cpu_list_id are important here as they are used in the filenames, e.g
```
        perf-record recorded CPU(s) 16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31 on host1 to files:
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list0.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.1.2023-07-13_11:32:42
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list0.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.2.2023-07-13_11:33:59
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list0.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.3.2023-07-13_11:35:17
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list0.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.4.2023-07-13_11:36:35
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list0.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.5.2023-07-13_11:37:53
    PASS 642_TestResult:
        perf-record recorded CPU(s) 16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31 on host1 to files:
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.1.2023-07-13_11:32:42
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.2.2023-07-13_11:34:00
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.3.2023-07-13_11:35:17
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.4.2023-07-13_11:36:35
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host1/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.5.2023-07-13_11:37:53
    PASS 643_TestResult:
        perf-record recorded CPU(s) 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15 on host2 to files:
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list0.cpus.0.1.2.3.4.5.6.7.8.9.10.11.12.13.14.15.data.1.2023-07-13_11:32:42
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list0.cpus.0.1.2.3.4.5.6.7.8.9.10.11.12.13.14.15.data.2.2023-07-13_11:34:00
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list0.cpus.0.1.2.3.4.5.6.7.8.9.10.11.12.13.14.15.data.3.2023-07-13_11:35:18
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list0.cpus.0.1.2.3.4.5.6.7.8.9.10.11.12.13.14.15.data.4.2023-07-13_11:36:35
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list0.cpus.0.1.2.3.4.5.6.7.8.9.10.11.12.13.14.15.data.5.2023-07-13_11:37:53
    PASS 644_TestResult:
        perf-record recorded CPU(s) 16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31 on host2 to files:
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.1.2023-07-13_11:32:44
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.2.2023-07-13_11:34:02
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.3.2023-07-13_11:35:20
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.4.2023-07-13_11:36:38
            /root/.lnst/lnst-logs/2023-07-13_11:30:46/SimpleNetworkTunableRecipe_match_0/linuxperf-data/host2/cpu_list1.cpus.16.17.18.19.20.21.22.23.24.25.26.27.28.29.30.31.data.5.2023-07-13_11:37:56
```